client/js/store/modules/game.js

```diff
@@ -269,6 +269,41 @@ export default {
         context.commit('setMyPNum', myPNum);
       }
     },
+    /**
+     * Updates gamestate to animate scuttle. First removes card from op hand
+     * and places it on top of player's point card, then waits 1s
+     * and updates complete game which will put both cards in the scrap
+     * @returns void
+     */
+    processScuttle(context, { game, playedCardId, targetCardId, playedBy }) {
+      // Update in one step if this player scuttled or if pNum is not set
+      if (!context.getters.player || playedBy === context.state.myPNum) {
+        context.dispatch('updateGameThenResetPNumIfNull', game);
+        return;
+      }
+      // Remove played card from opponent hand and temporarily add to its targets attachments
+      const playedCardIndex = context.getters.opponent.hand.findIndex(
+        (card) => card.id === playedCardId
+      );
+      const targetCardIndex = context.getters.player.points.findIndex(
+        (card) => card.id === targetCardId
+      );
+
+      // Update game in one-step if mvoed cards are not found
+      if (playedCardIndex === undefined || targetCardIndex === undefined) {
+        context.dispatch('updateGameThenResetPNumIfNull', game);
+        return;
+      }
+
+      const [playedCard] = context.getters.opponent.hand.splice(playedCardIndex, 1);
+      const targetCard = context.getters.player.points[targetCardIndex];
+      targetCard.scuttledBy = playedCard;
+
+      // Finish complete update of the game state after 1s
+      setTimeout(() => {
+        context.dispatch('updateGameThenResetPNumIfNull', game);
+      }, 1000);
```

Comment:
Maybe it's just me but this whole function feels complicated. Are we not able to just track those new ids and use coordinates to animate a single absolute positioned element across the gameboard or does it have to be attached to use the transitions?

Reply:
Agreed it feels complicated and generally smells. That said imo a coordinate-based system would likely be an overall improvement but a more complicated one. I'm definitely in favor of re-examining the whole transition system accordingly, so I'm thinking separate issue.

That said the vast majority of the complexity here is due to the 2-stage nature of the intended animation itself:
1) the opponent's point card fades out downwards while the scuttling card simultaneously fades in from above on top of your point card, THEN
2) your point card with the scuttling card fades out to the left (towards scrap)

I'm open to suggestions for how to simplify. Would love to iscuss. My thought is it warrants a holistic second pass after this PR

Reply:
Wrote up https://github.com/cuttle-cards/cuttle/issues/233 to track the issue of replacing the transition/animation system with an absolute coordinate one. I think doing so will be complicated and probably require a senior engineer, but it would be a significant improvement